draft-ietf-quic-transport.md

```diff
@@ -1714,9 +1798,9 @@ state to draining, but it otherwise has no impact.
 An endpoint could receive packets from a new source address, indicating a
 connection migration ({{migration}}), while in the closing period. An endpoint
 in the closing state MUST strictly limit the number of packets it sends to this
-new address as though the address were not validated (see {{migrate-validate}}).
-A server in the closing state MAY instead choose to discard packets received
-from a new source address.
+new address as though the address were not validated (see
```

Comment:
I know this isn't new text, but why "as though the address were" rather than "since the address is not"?

Reply:
Good point, fixed.